libs/designsystem/form-field/src/input/input.component.ts

```diff
@@ -24,7 +26,23 @@ export enum InputSize {
   styleUrls: ['./input.component.scss'],
   template: '',
 })
-export class InputComponent implements OnChanges {
+export class InputComponent implements OnChanges, OnInit {
+  constructor(private elementRef: ElementRef<HTMLInputElement>) {}
+
+  ngOnInit(): void {
+    // The native input value is emitted here to make sure that
+    // the input counter component receives the value onInit,
```

Comment:
Consider readability:
```suggestion
    // the InputCounterComponent receives the value onInit,
```